src/WobblySpinner.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { propTypes, defaultProps } from "./WobblySpinner.props";
+import * as wobblySpinnerSheet from "./wobblySpinnerSheet";
+
+function mountSheet(props) {
+    var { key, className } = wobblySpinnerSheet.mount(props);
+    this._key = key;
+    this._className = className;
+}
+
+export default class WobblySpinner extends React.Component {
+    static propTypes = propTypes
+    static defaultProps = defaultProps
+
+    componentWillMount() {
+        this::mountSheet(this.props);
```

Comment:
What's the reason for using a very experimental es feature? When I see a stand alone function outside a class it appears to be static, which in this case it's not. It's part of the class? Why not just put it inside the class. Then you just get the `this` context for free.